src/core/Paperwork/US_ND/EarlyTermination/Transformer.ts

```diff
@@ -41,13 +41,16 @@ export const transform = (
   return {
     clientName: client.displayName,
     judgeName,
-    convictionCounty,
+    convictionCounty: convictionCounty?.replaceAll("_", " ") ?? "",
+    judicialDistrictCode: judicialDistrictCode?.replaceAll("_", " ") ?? "",
     priorCourtDate,
     probationExpirationDate,
-    sentenceLengthYears,
-    crimeNames,
+    sentenceLength: `${sentenceLengthYears} year${
+      sentenceLengthYears === "1" ? "" : "s"
+    }`,
```

Comment:
There's a `pluralize()` helper function in `formatStrings.ts`

```suggestion
    sentenceLength: pluralize(sentenceLengthYears, "year"),
```